pom.xml

```diff
@@ -2616,6 +2616,11 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>io.airlift</groupId>
+        <artifactId>aircompressor</artifactId>
```

Comment:
just curious why we need to bring a new dependency for this: where is the old dependency (`aircompressor` 0.25) come from?

Reply:
Basically, it came from Apache ORC library.

Reply:
Here
- https://mvnrepository.com/artifact/org.apache.orc/orc-core/1.9.2

Reply:
Also, Parquet also has some dependency.
- https://mvnrepository.com/artifact/org.apache.parquet/parquet-hadoop/1.13.1

However, I'm not sure about Parquet part. This PR aims to override it from Apache Spark side.

Reply:
I see, thanks